ietf-wdm-tunnel.yang

```diff
@@ -78,17 +76,17 @@ module ietf-wdm-tunnel {
   identity lower-first-wavelength-assignment {
     base l0-types:wavelength-assignment;
     description
-      "Assign first available wavelength starting from the
-       lowest frequency towards the highest frequency in
-       the allowable frequency range.";
+      "Allocate wavelengths in ascending order, beginning from the
+       lowest frequency and progressing toward the highest frequency
+       within the permissible frequency range.";
```

Comment:
1. Should this identity be move to L0 types?

   See: https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/issues/92

2. Is there a reference where this wavelength assignment is defined?

Reply:
issue resolved by https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/96